Sasquatch/Sasquatch/ViewControllers/Sections/CommonSchemaPropertiesTableSection.swift

```diff
@@ -60,8 +60,22 @@ class CommonSchemaPropertiesTableSection : SimplePropertiesTableSection {
     }
   }
   
+  // Note down current target for using in End edit events
+  override func recordCurrentTarget(sender: UITextField!) {
+    let curCell = sender.superview!.superview as! MSAnalyticsPropertyTableViewCell
+    curCell.curTarget = transmissionTargetSelectorCell?.selectedTransmissionTarget() ?? ""
+  }
+    
+// Add below method since tap out of the key textField will crash the app, that's a bad experience
```

Comment:
Please format the code.

Reply:
Hum I checked it and made some format changes, please tell me if you found not meet your expectations, thanks~